doc/source/user_guide/options.rst

```diff
@@ -482,6 +482,11 @@ plotting.backend                        matplotlib   Change the plotting backend
                                                      like Bokeh, Altair, etc.
 plotting.matplotlib.register_converters True         Register custom converters with
                                                      matplotlib. Set to False to de-register.
+styler.sparsify_index                   True         "Sparsify" MultiIndex display for rows
```

Comment:
we had a discussion a while back about nesting display.styler..... (e.g. as we do display.html), but maybe this was to *move* .html to the top level, this is prob better

consider styler.sparse.index

Reply:
i think having its own context is useful for what follows. I have changed to your consideration.